torch/csrc/jit/script/init.cpp

```diff
@@ -824,12 +824,12 @@ void initJitScriptBindings(PyObject* module) {
           })
       .def(
           "_python_print",
-          [](Module& self, bool enforce_importable) {
+          [](Module& self) {
             std::ostringstream ss;
             std::vector<at::Tensor> tensors;
-            PythonPrint(ss, self, tensors, enforce_importable);
+            PythonPrint(ss, self, tensors, true);
```

Comment:
If this is hardcoded to true here, why does it even exist in the PythonPrint method? Maybe we should just remove it altogether?

Reply:
Looks like it's used for the `ScriptModule.code` property to allow Python ops to be pretty-printed